source/Ship.cpp

```diff
@@ -3885,6 +3885,141 @@ void Ship::AddOutfit(const Outfit *outfit, int count)
 
 
 
+void Ship::AddHardpoint(DataNode data) {
```

Comment:
Accept a const reference and call it `node`.

Reply:
will do this as soon as it got tested

Reply:
~~Not taking a reference means it might not actually work.~~

Reply:
ok, I do it now

Reply:
Change the name in the header file as well.

Reply:
yups